app/factories/common.py

```diff
@@ -11,6 +11,7 @@
 float_ = '1.23456789'
 timezone_ = 'UTC'
 slug_ = 'camp-trip-retreat'
+fee_ = '3.14'
```

Comment:
Added since currency involves only two decimal places floating point numbers , so **float_** can't be used.
( suggested by SaptakS [here](https://github.com/fossasia/open-event-orga-server/pull/3866#discussion_r125050533) )